templates/console.tmpl

```diff
@@ -73,20 +73,20 @@
             
             var requestData = $(this).data("request-data");
 
-            $("#requestHeaders").html($("<pre/>").addClass("pre-scrollable").text(JSON.stringify(requestData.requestHeaders, null, 4))[0].outerHTML);
+            $("#requestHeaders").html($("<pre/>").addClass("pre-scrollable w-100").text(JSON.stringify(requestData.requestHeaders, null, 4))[0].outerHTML);
 
             if ("Content-Type" in requestData.requestHeaders && requestData.requestHeaders["Content-Type"][0] == "application/json") {
-                $("#requestBody").html($("<pre/>").addClass("pre-scrollable").text(JSON.stringify(JSON.parse(atob(requestData.requestBody)), null, 4))[0].outerHTML);
+                $("#requestBody").html($("<pre/>").addClass("pre-scrollable w-100").text(JSON.stringify(JSON.parse(atob(requestData.requestBody)), null, 4))[0].outerHTML);
             } else {
-                $("#requestBody").html($("<pre/>").addClass("pre-scrollable").text(atob(requestData.requestBody))[0].outerHTML);
+                $("#requestBody").html($("<pre/>").addClass("pre-scrollable w-100").text(JSON.stringify(JSON.parse(atob(requestData.requestBody)), null, 4))[0].outerHTML);
```

Comment:
Was this intentional or copypasta? I think this will actually break the response/request data fields

Reply:
Ok I will check it again.
I just copyied this method, because I checked in jsfiddle that it will format json in requestBody field.
How this will break the response/request data fields ?
I don't understand your reason. Is there any typo error, or general error.

Reply:
Those fields aren't necessarily JSON, so the parse will possibly fail